cmd/earthly/subcmd/github_cmds.go

```diff
@@ -92,14 +152,97 @@ func (a *Github) actionAdd(cliCtx *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	err = cloudClient.SetGithubToken(cliCtx.Context, a.Org, a.GHOrg, a.GHRepo, a.GHToken)
+	err = cloudClient.CreateGHAIntegration(cliCtx.Context, a.Org, a.GHOrg, a.GHRepo, a.GHToken)
 	if err != nil {
 		return fmt.Errorf("error found running github add: %w", err)
 	}
 	a.cli.Console().Printf("GitHub integration successfully created")
 	return nil
 }
 
+func (a *Github) actionRemove(cliCtx *cli.Context) error {
+	a.cli.SetCommandName("githubRemove")
+	if a.Org == "" {
+		if a.cli.OrgName() == "" {
+			return fmt.Errorf("coudn't determine Earthly organization")
+		}
+		a.Org = a.cli.OrgName()
+	}
+	cloudClient, err := helper.NewCloudClient(a.cli)
+	if err != nil {
+		return err
+	}
+	err = cloudClient.RemoveGHAIntegration(cliCtx.Context, a.Org, a.GHOrg, a.GHRepo)
+	if err != nil {
+		return fmt.Errorf("error found running github remove: %w", err)
+	}
+	a.cli.Console().Printf("GitHub integration successfully removed")
+	return nil
+}
+
+func (a *Github) actionList(cliCtx *cli.Context) error {
+	a.cli.SetCommandName("githubList")
+	if a.Org == "" {
+		if a.cli.OrgName() == "" {
+			return fmt.Errorf("coudn't determine Earthly organization")
+		}
+		a.Org = a.cli.OrgName()
+	}
+	cloudClient, err := helper.NewCloudClient(a.cli)
+	if err != nil {
+		return err
+	}
+	integrations, err := cloudClient.ListGHAIntegrations(cliCtx.Context, a.Org)
+	if err != nil {
+		return fmt.Errorf("error found running github list: %w", err)
+	}
+	if a.printJSON {
+		if err := a.printIntegrationsJSON(integrations); err != nil {
+			return err
+		}
+	} else {
+		if err := a.printIntegrationsTable(integrations); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (a *Github) printIntegrationsJSON(integrations *pb.ListGHAIntegrationsResponse) error {
+	jsonInts := make([]integrationJSON, len(integrations.Integrations))
+	for i, integration := range integrations.Integrations {
+		jsonInts[i] = integrationJSON{
+			GithubOrgName:  integration.GithubOrgName,
+			GitHubRepoName: integration.GithubRepoName,
+			CreatedBy:      integration.CreatedBy,
+			CreatedAt:      integration.CreatedAt.AsTime().Format(time.DateTime),
+		}
+	}
+	b, err := json.MarshalIndent(jsonInts, "", "  ")
+	if err != nil {
+		return fmt.Errorf("failed to marshal json: %w", err)
+	}
+	a.cli.Console().Printf(string(b))
```

Comment:
what if `b` contains a `%` character? If you were calling a function like `Print` or `Println` this would be fine, but `Printf` expects a formatting string as the first argument.
```suggestion
	a.cli.Console().Printf("%s", string(b))
```

or simply `a.cli.Console().Printf("%s", b)`

also, do you need a `\n`?